src/parser/serializer.ts

```diff
@@ -231,10 +231,9 @@ function serializeObjectId(buffer: Uint8Array, key: string, value: ObjectId, ind
   // Write the type
   buffer[index++] = constants.BSON_DATA_OID;
   // Number of written bytes
-  const numberOfWrittenBytes = ByteUtils.encodeUTF8Into(buffer, key, index);
+  index += ByteUtils.encodeUTF8Into(buffer, key, index);
```

Comment:
No **real** functional impact here, Just wanted to be somewhat consistent with line **239** below and save the extra `Number` allocation in the hopes it would reduce GC load. I doubt this has any measurable effect though. 

```ts
// line 239
index += value.serializeInto(buffer, index);
```